### PR TITLE
Upgrade to new affjax

### PR DESCRIPTION
**Pull request by i-am-the-slime**
Great library, thank you!

I'm not sure about the changes to bower since I only really use spago. But the change was quite easy.

#5 and #4 seem to be fixed by this.

### CODE REVIEW

**Comment by i-am-the-slime** [comment]
Please note that I added my own fork to the spago package set so you probably don't want to merge it like this.

**Comment by newlandsvalley** [comment]
Many thanks, Mark.  Yes - I've been resisting moving to Spago for some reason until the new registry is ready and bower and pulp have usually served me well in the past.  I'll probably cherry-pick the affjax update though!

File: packages.dhall [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+       [ "prelude"
+       , "lists"
+       , "string-parsers"
+       , "integers"
+       , "unicode"
+       , "generics-rep"
+       , "signal"
+       ]
+    , repo =
+       "https://github.com/i-am-the-slime/purescript-midi.git"
```
Comment (newlandsvalley) [inline packages.dhall]: I've now added spago to my purescript-midi.  Could you now point to my repo (as you suggested)?

File: src/Audio/SoundFont.purs [hunk 2]
```diff
@@ -123,7 +122,7 @@ loadInstrument maybeLocalDir instrumentName = do
   res <- request $ defaultRequest
     { url = url, method = Left GET, responseFormat = ResponseFormat.string }
 
-  case res.body of
+  case res <#> _.body of
```
Comment (newlandsvalley) [inline src/Audio/SoundFont.purs:125]: I'm intrigued why you made this change.

**Review by newlandsvalley — CHANGES_REQUESTED** [review]

File: src/Audio/SoundFont.purs (hunk 2 above)
Comment (i-am-the-slime) [reply src/Audio/SoundFont.purs:125]: This is to accommodate the API change in Affjax. Previously `res.body` was an `Either`. Now `res` is an `Either` but `res.body` is just a `String`.

This particular line is equivalent to `_.body <$> res` or `map _.body res` or `map (\x -> x.body) res` or `mapFlipped res _.body` which is really what `<#>` is. An infix version of `mapFlipped`.

Comment (newlandsvalley) [reply src/Audio/SoundFont.purs:125]: Ah - of course.  Many thanks/

**Comment by i-am-the-slime** [comment]
I guess the next step would be to add everything to the package set!

**Review by newlandsvalley — APPROVED** [review]
Very many thanks for doing this, Mark.

**Comment by newlandsvalley** [comment]
Both this and midi now live in the package set.

**Pull request merged**